devbox/src/cmdproxy/ProxyServer.scala

```diff
@@ -34,7 +34,7 @@ object Request {
  * the end of the output stream
  */
 class ProxyServer(dirMapping: Seq[(os.Path, os.RelPath)],
-                  port: Int = ProxyServer.DEFAULT_PORT)
+                  port: Int)
                  (implicit logger: FileLogger) {
 
   // this may throw when binding if the socket is used, but for the moment we just assume there is no other
```

Comment:
I think this comment may no longer apply

Reply:
ReliableAgent has some logic to restart the server socket, in which case we still give it a non-0 port number because we need to bind to the same port which may have already been used for the tunnel.